python/dataproc_templates/snowflake/README.md

```diff
@@ -0,0 +1,133 @@
+## 1. Snowflake To GCS
+
+General Execution:
+
+```
+export GCP_PROJECT=<gcp-project-id>
+export REGION=<gcp-project-region>
+export GCS_STAGING_LOCATION=<gcs-bucket-staging-folder-path>
+export SUBNET=<gcp-project-dataproc-clusters-subnet>
```

Comment:
Add the export JARS="gs://<...>/spark-snowflake_2.12-2.10.0-spark_3.1.jar,gs://<...>/snowflake-jdbc-3.13.14.jar"